boardfarm/lib/bft_logging.py

```diff
@@ -238,9 +238,13 @@ def add_to_combined_list(log, name, combined_list=combined_list):
                     line = line[1:]
                 if line.startswith(' ['):
                     line = line[1:]
-                ts, text = line.split(']', 1)
+                    ts, text = line.split(']', 1)
+                    timestamp = float(ts[1:-1])
+                else:
+                    line = text
```

Comment:
Did you test this?

It should be `text = line` of course...

Reply:
> Did you test this?
> 
> It should be `text = line` of course...

For some reason, boardfarm still works with that...
But you're right, I'll fix that.